src/javascript/app/pages/resources/asset_index/asset_index.ui.js

```diff
@@ -29,6 +30,8 @@ const AssetIndexUI = (() => {
     const populateTable = () => {
         if (!active_symbols || !asset_index) return;
 
+        if (!asset_index.length) $('#empty-asset-index').setVisibility(1);
```

Comment:
do we need to return after this?

Reply:
return here causes the showLoadingImage fire and the loader is visible constantly

Reply:
so we should hide the loader and return, doesn't make sense to do all these actions to get asset index data and build the table when there's no data to show